TradeItIosTicketSDK2/TradeItPortfolioEquityPositionPresenter.swift

```diff
@@ -16,11 +16,8 @@ class TradeItPortfolioEquityPositionPresenter: TradeItPortfolioPositionPresenter
         return symbol
     }
 
-    func getQuantity() -> Float? {
-        guard let quantity = self.position?.quantity
-            else { return 0 }
-        
-        return quantity.floatValue
+    func getQuantity() -> NSNumber? {
+        return self.position?.quantity
```

Comment:
I removed the default to 0, because somewhere else we use the getQuantity method and we want to return N/A if it is not available